docs/articles/test-workflows-content.mdx

```diff
@@ -187,6 +187,16 @@ spec:
 
 :::
 
+### Working Directory
```

Comment:
Perhaps call it "Working Directory Resolution" and move it up to level 2  before the Git section?

Reply:
done